mixlib-install.gemspec

```diff
@@ -24,4 +24,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.3"
   spec.add_development_dependency "pry"
+  spec.add_development_dependency "climate_control"
```

Comment:
Looks like this gem brings in ActiveSupport [as a dependency](https://github.com/thoughtbot/climate_control/blob/master/climate_control.gemspec#L20) to make things thread-safe. Do we really need that baggage here? 

We can bring in some simplified version of [these](https://github.com/thoughtbot/climate_control/blob/master/lib/climate_control/modifier.rb) functions into our spec_helper and we should be good to go.